logic.py

```
@@ -453,6 +531,15 @@ def _is_candidate_valid(col):
             else:
                 multi_df[c] = df_aligned[c].apply(clean_numeric_value)
         
+        # ✅ NEW: Add interaction terms if specified
+        if interaction_pairs:
+            try:
+                from interaction_lib import create_interaction_terms
+                multi_df, int_meta = create_interaction_terms(multi_df, interaction_pairs, mode_map)
+                logger.info(f"✅ Added {len(int_meta)} interaction terms to logistic multivariate model")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)